ccd-definition/ComplexTypes/LitigationFriend.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "ID": "LitigationFriend",
+    "ListElementCode": "required",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Is the claimant a child?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "LitigationFriend",
+    "ListElementCode": "fullName",
+    "FieldType": "Text",
+    "ElementLabel": "Litigation friend's full name",
+    "FieldShowCondition": "required = \"Yes\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "LitigationFriend",
+    "ListElementCode": "differentAddress",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Are the claimant and litigation friend address the same?",
+    "FieldShowCondition": "required = \"Yes\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "LitigationFriend",
+    "ListElementCode": "address",
```

Comment:
can we change this to litigationFriendPrimaryAddress?

Reply:
Due to complex type context, not needed. Why primaryAddress over address?

Reply:
Addresses should be called out specifically for their purpose. Quite often someone sees 'address' and will use it for whatever reason they choose. This is problematic when any old address is used for something like correspondence. It's also consistent with the litigant details.